zvariant/src/value.rs

```diff
@@ -985,6 +985,7 @@ mod tests {
                 (@mmn nothing, @mmmn just nothing))"
         );
 
+        #[cfg(unix)]
```

Comment:
Why didn't our CI pick it up?

Reply:
`cargo test --package zbus --features windows-gdbus` ?

Reply:
`cargo test --features=zbus/windows-gdbus` should solve this